components/scream/src/mct_coupling/atm_comp_mct.F90

```diff
@@ -55,7 +55,8 @@ subroutine atm_init_mct( EClock, cdata, x2a, a2x, NLFilename )
                                   scream_init_atm
     use scream_cpl_indices, only: scream_set_cpl_indices, num_cpl_exports, &
                                   num_cpl_imports, num_scream_imports, &
-                                  scr_names_x2a, scr_names_a2x, index_x2a, index_a2x, vec_comp_x2a, vec_comp_a2x
+                                  scr_names_x2a, scr_names_a2x, index_x2a, index_a2x, vec_comp_x2a, vec_comp_a2x, &
+                                  can_be_exported_during_init
```

Comment:
Idea: can't we figure out (from scream) the fields that must be skipped during the init phase just by looking at the time stamp? Meaning, if we init a field (either during IC setup or during an atm proc init phase, like p_mid/p_int in Homme), we _should_ set the time stamp of the field to `t0`. If an export field has time stamp still set to invalid, then it cannot be exported (obv, during a _normal_ export, we must have no invalid field).

Reply:
On a second thought, this would blindly skip any field that has invalid timestamp at init phase, but we might want to be able to _ensure_ that the ones that _should_ be exported are exported. So probably not a great idea? I was trying to see if we can get the AD to automatically do stuff, without making the F90 interface heavier.

Reply:
Well, in general, if a quantity is not initialized by scream, it can't be exported. So simply looking at the time stamp might be a valid approach (perhaps we could print a msg to screen when we skip a field during the init phase).

However, this would be a bit cumbersome. We can punt on this, and polish later, if we find a decent solution.